fetch-utils.js

```diff
@@ -15,7 +41,7 @@ export function checkAuth() {
 
 export function redirectIfLoggedIn() {
```

Comment:
You can also use a ternary here to redirect the user: getUser() ? location.replace('./shopping-list) : window.location.href = '../'